app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -653,14 +653,14 @@ private String getIVText() {
                 returnVal += "<br>"; //breakline
                 //for following stage evolution (example, dratini - dragonair - dragonite)
                 //if the current evolution has another evolution calculate its range and break
-                for (Pokemon nextEvo : evolution.evolutions) {
+                if(evolution.evolutions.size()>0) {
```

Comment:
Sorry but I can't waste time on manual code formatting. If this pull request is declined - so be it. If not - a separate formatting commit is always welcome.